apps/meteor/client/views/admin/apps/BundleChips.tsx

```diff
@@ -15,19 +15,28 @@ type BundleChipsProps = {
 
 const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
 	const t = useTranslation();
-
+	const bundleRef = useRef<Element>();
+	const [isHovered, setIsHovered] = useState(false);
 	return (
 		<>
-			{bundledIn.map((bundle, index) => (
-				<Tag
-					key={index}
-					title={t('this_app_is_included_with_subscription', {
-						bundleName: bundle.bundleName,
-					})}
-					variant='primary'
-				>
-					{bundle.bundleName}
-				</Tag>
+			{bundledIn.map((bundle) => (
+				<Fragment key={bundle.bundleId}>
+					<Box ref={bundleRef} onMouseEnter={(): void => setIsHovered(true)} onMouseLeave={(): void => setIsHovered(false)}>
+						<Tag variant='featured'>{bundle.bundleName}</Tag>
+					</Box>
+					<PositionAnimated
```

Comment:
just passing a title to the component is enough to dispatch a tooltip!

Reply:
Solved